packages/schema/src/types/nuxt.ts

```diff
@@ -38,6 +38,8 @@ export interface ResolvedNuxtTemplate<Options = Record<string, any>> extends Nux
   dst: string
 }
 
+export type NuxtTypeTemplate<Options = Record<string, any>> = Omit<NuxtTemplate<Options>, 'write'>
```

Comment:
What do you think about instead having `write?: true` to preserve general compatibility?

Reply:
I was thinking about it too. In purpose of global compatibility we can, of course, keep the param, but set it to `true`